public/javascripts/ng-apps.js

```diff
@@ -1,17 +1,21 @@
-var springInitiative = angular.module('springInitiative', ['ngRoute']);
+var springInitiative = angular.module('springInitiativeApp', ['ui.router']);
 
-springInitiative.config(function($routeProvider) {
-  $routeProvider
-    .when('/student', {
-      templateUrl: 'views/student.html',
+springInitiative.config(function($stateProvider, $urlRouterProvider) {
+
+  $urlRouterProvider.otherwise('/login');
```

Comment:
Also though, / isn't redirecting to anywhere but login right now so maybe we should fix that here as well.  (Like when you're logged in / isn't redirecting as far as i know)